core/src/main/scala/com/salesforce/op/stages/impl/classification/OpNaiveBayes.scala

```diff
@@ -103,10 +103,10 @@ class OpNaiveBayesModel
 ) extends OpProbabilisticClassifierModel[NaiveBayesModel](
   sparkModel = sparkModel, uid = uid, operationName = operationName
 ) {
-  @transient lazy val predictRawMirror = reflectMethod(getSparkMlStage().get, "predictRaw")
-  @transient lazy val raw2probabilityMirror = reflectMethod(getSparkMlStage().get, "raw2probability")
-  @transient lazy val probability2predictionMirror =
-    reflectMethod(getSparkMlStage().get, "probability2prediction")
+  @transient lazy val predictRawMirror = getSparkOrLocalMethod("predictRaw", "predictRaw")
+  @transient lazy val raw2probabilityMirror = getSparkOrLocalMethod("raw2probability", "rawToProbability")
```

Comment:
nit: any way to align these names to `rawToProbabilitySpark` / `rawToProbabilityLocal`? same for `probabilityToPrediction`

Reply:
So these are the names on the underlying spark and mleap classes. We are using reflection to access private methods (a dangerous hack but...) so we dont have any control over the names.